Rakefile

```diff
@@ -11,15 +11,6 @@ task :test => ['test:newrelic']
 namespace :test do
   desc "Run all tests"
   task :all => %w[newrelic multiverse all_compatible_envs]
-
-  begin
-    require 'test_bisect'
-    TestBisect::BisectTask.new do |t|
-      t.test_task_name = 'test:newrelic'
-    end
-  rescue LoadError
-  end
```

Comment:
We don't currently use `test_bisect` (untouched since 2013), so let's stop rescuing its load error on every single Rake task run.

Reply:
If we need functionality like this in the future, I suggest we use the Minitest Bisect gem - https://rubygems.org/gems/minitest-bisect